frontend/src/components/SamplesList.tsx

```diff
@@ -90,26 +96,23 @@ export default function SamplesList({
   const [showUpdateModal, setShowUpdateModal] = useState(false);
   const [changes, setChanges] = useState<SampleChange[]>([]);
   const [editMode, setEditMode] = useState(true);
-  const [alertContent, setAlertContent] = useState(ROW_LIMIT_ALERT_CONTENT);
-  const [rowCount, setRowCount] = useState(0);
+  const [alertContent, setAlertContent] = useState(MAX_ROWS_SCROLLED_ALERT);
+  const [sampleCount, setSampleCount] = useState(0);
 
   const gridRef = useRef<AgGridReactType>(null);
   const params = useParams();
 
   useEffect(() => {
     gridRef.current?.api?.showLoadingOverlay();
-    async function refetchSearchVal() {
-      await refetch({
-        where: refetchWhereVariables(parsedSearchVals),
-      });
-    }
-    refetchSearchVal().then(() => {
```

Comment:
Feels good to actually reduce a bit of code here. `refetch` is not an async function so we didn't need to `async/await` it. Rather, it's a regular function that returns a promise, which is slightly different.